R/calendar_range.R

```diff
@@ -0,0 +1,114 @@
+#' Create Semantic UI Calendar Range
+#'
+#' This is a wrapper for creating calendar ranges using Semantic UI calendars.
+#' It creates a form composed of two calendars. The selected range values are
+#' available under \code{input[[input_id]]}.
+#'
+#' @details The Semantic UI calendar range automatically pops up the end range
+#' calendar when changing the start range. Therefore events are only sent on
+#' changes done in the end range calendar.
+#'
+#' @param input_id Input name. Reactive value is available under \code{input[[input_id]]}.
+#' @param type Select from \code{'year'}, \code{'month'}, \code{'date'} and \code{'time'}.
+#' @param start_value Initial value of the calendar defining the start of the range.
+#' @param end_value Initial value of the calendar defining the end of the range.
+#' @param start_placeholder Text visible in the start calendar input when nothing is inputted.
+#' @param end_placeholder Text visible in the end calendar input when nothing is inputted.
+#' @param min Minimum allowed value in both calendars.
+#' @param max Maximum allowed value in both calendars.
```

Comment:
I suggest having the same order of the arguments as in the calendar() function: id, value, placeholder, type, min, max